apps/authenticator/lib/sign_in/commands/inputs/resource_owner.ex

```diff
@@ -40,6 +41,7 @@ defmodule Authenticator.SignIn.Inputs.ResourceOwner do
     |> cast(params, @required ++ @optional)
     |> validate_length(:username, min: 1)
     |> validate_length(:password, min: 1)
+    |> validate_format(:otp, ~r(\d{4,6}))
```

Comment:
I miss a helper over here. `validate_otp/2` would be lovely

Reply:
I don't know. It will probably only be used here.